### ISSUE TITLE
Expose `MonetizationCurrencyAmount`

### ISSUE THREAD

**Issue by sidvishnoi** [interledger/web-monetization-extension#501]
Spec says it's exposed on Window.
Also, make sure `amountSent` is an instance of `MonetizationCurrencyAmount`.

**Comment by sidvishnoi** [comment]
On a second thought, I think it should not be exposed to Window (spec bug). If `MonetizationCurrencyAmount` maps directly to `PaymentCurrencyAmount`, why it's an interface and not a dictionary like `PaymentCurrencyAmount`?

**Comment by MichaelKohler** [comment]
> If MonetizationCurrencyAmount maps directly to PaymentCurrencyAmount, why it's an interface and not a dictionary like PaymentCurrencyAmount?

Yeah, I agree as well. I do not see any point in exposing this to `Window` as long as its only use is for the event.

This then also makes me wonder if it is needed at all. The `amountSent` member of the [MonetizationEventInit](https://webmonetization.org/specification/#ref-for-dom-monetizationeventinit-2) dictionary already directly references [PaymentCurrencyAmount](https://www.w3.org/TR/payment-request/#dom-paymentcurrencyamount). I don't know what the best practice is here, but maybe we could just remove the definition within the Web Monetization spec?

Can you transfer this issue to the spec repo? Or should I create a new one?

**Comment by MichaelKohler** [comment]
> But I'm considering if it can be an internal interface - not exposed to Window, and if that can cause issues.

Unfortunately this won't work. We could use [LegacyNoInterfaceObject](https://webidl.spec.whatwg.org/#LegacyNoInterfaceObject), however it [should not be used for new APIs](https://webidl.spec.whatwg.org/#js-legacy-extended-attributes).

That being said, we do not actually have a constructor for this, so this already is correct behavior.

I'm gonna create a PR to add it with the same semantics in the polyfill.
